lib/src/compile.ts

```diff
@@ -66,13 +71,20 @@ export async function compileString(options: {
 function newCompileRequest(options: {
   path: string;
   sourceMap: boolean;
+  includePaths: string[];
 }): proto.InboundMessage.CompileRequest {
   // TODO(awjin): Populate request with importer/function IDs.
 
   const request = new proto.InboundMessage.CompileRequest();
   request.setPath(options.path);
   request.setSourceMap(options.sourceMap);
 
+  for (const path of options.includePaths) {
+    const importer = new proto.InboundMessage.CompileRequest.Importer();
+    importer.setPath(path);
```

Comment:
Here and below: resolve `path` before passing it to the importer.

Reply:
I'm guessing you wanted me to Path.resolve the `path` string before it's passed into `importer.setPath(...)`.